index.html

```diff
@@ -281,6 +281,11 @@ <h4>Game streaming</h4>
              latency for audio, video and data under varying network conditions. This may
              include tweaking of transport parameters for both media and data.</td>
         </tr>
+	<tr>
+           <td>N36</td>
+           <td>An application that is only receiving but not sending media can operate
+	       without prompts for camera and microphone permissions.</td>
```

Comment:
The prompt is a symptom. I think we should clarify it's the requirement of camera or microphone access itself that is problematic (a permission escalation that is bad for users as well as apps). This is also comes up a lot with data channels. How about:
```suggestion
           <td>An application that is only receiving but not sending media or data can operate
	       without access to camera or microphone.</td>
```

Reply:
> The prompt is a symptom. I think we should clarify it's the requirement of camera or microphone access itself that is problematic (a permission escalation that is bad for users as well as apps). This also comes up a lot with data channels. How about:

@jan-ivar  Thank you for the review. The proposed changes look good to me. Once we will get other reviewer's opinion, will commit the changes.